Sanitizers

satoren opened the pull request:
test

satoren committed:
enable sanitizers on tests

ibc commented on .github/workflows/mediasoup-node.yaml:
```diff
@@ -11,12 +11,23 @@ concurrency:
+          - os: ubuntu-22.04
+            node: 20
+            meson_args: '-Db_sanitize=address'
+            asan_option: 'detect_leaks=0' # Disable leak detection because detect memory leaks in flatbuffers tool.
```
What is the leak detection issue in flatbuffers?

satoren committed:
fix asan error for new-delete-type-mismatch (#1411)

ibc committed:
TcpConnectionHandle.cpp: properly close handle

This is the same change as done in PR #1411 but here in `TcpConnectionHandle.cpp`.

ibc committed:
Update NPM deps

Lynnworld committed:
fix Simulcast IncreaseLayer bug when producer score is zero (#1410)

ibc committed:
cosmetic

ibc committed:
Fix issue #1374

ibc committed:
Update npm deps

ibc committed:
Update NPM dev deps

DavidNegro committed:
Add support for playout-delay extension (#1412)

ibc committed:
FuzzerRtpPacket: use PlayoutDelayExtension

ibc committed:
3.14.8
